pages/common/ab.md

```diff
@@ -1,24 +1,24 @@
 # ab
 
-> Apache Benchmarking tool. The simplest tool to perform a load testing.
-> More information: <https://httpd.apache.org/docs/2.4/programs/ab.html>.
+> Apache HTTP server benchmarking tool.
+> More information: <https://httpd.apache.org/docs/current/programs/ab.html>.
 
-- Execute 100 HTTP GET requests to a given URL:
+- Execute 100 HTTP GET requests to a URL:
 
 `ab -n {{100}} {{url}}`
 
-- Execute 100 HTTP GET requests, processing up to 10 requests concurrently, to given URL:
+- Execute 100 HTTP GET requests, in concurrent batches of 10, to a URL:
 
 `ab -n {{100}} -c {{10}} {{url}}`
 
-- Use keep alive:
+- Execute 100 HTTP POST requests to a given URL, using a JSON payload from a file:
```

Comment:
Since you removed it from the other examples I assume that shouldn't be there.
```suggestion
- Execute 100 HTTP POST requests to a URL, using a JSON payload from a file:
```

Reply:
Definitely.  Oversight 👍🏻